packages/config-transformer/tests/Converter/ConfigFormatConverter/Source/InvokableFactory.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Symplify\ConfigTransformer\Tests\Converter\ConfigFormatConverter\Source;
+
+/**
+ * @see https://symfony.com/doc/current/service_container/factories.html#invokable-factories
+ */
+final class InvokableFactory
```

Comment:
I don't know if this class is required, probably not (for other factories/configurators fake classes are used).